requirements.txt

```diff
@@ -8,4 +8,5 @@ optax~=0.2.2
 ply~=3.11
 PyYAML~=6.0.1
 tqdm~=4.66.2
-wandb~=0.17.2
\ No newline at end of file
+wandb~=0.17.2
+nltk
```

Comment:
Please add the library version so that we can avoid problems with API changes in other versions